ci-operator/jobs/openshift/odo/openshift-odo-main-periodics.yaml

```diff
@@ -0,0 +1,269 @@
+periodics:
+- agent: kubernetes
+  cluster: build01
+  cron: 0 */6 * * *
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: main
+    org: openshift
+    repo: odo
+  labels:
+    ci-operator.openshift.io/prowgen-controlled: "true"
+    ci-operator.openshift.io/variant: v4.5
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-odo-main-v4.5-integration-e2e-periodic
```

Comment:
@mohammedzee1000 I think you missed adding `reporter_config` for periodic jobs https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/odo/openshift-odo-master-periodics.yaml#L17-L27
Could you please add it for all the periodic jobs?

Reply:
Added, will push shortly

Reply:
Still its not reflected 🙁 . Actually when we do `make jobs` it removes the extra config we need. So it needs to be added once we are done with `make jobs`

Reply:
Ahh my bad  I thought you already pushed the changes.